lib/embedded_ansible.rb

```diff
@@ -0,0 +1,23 @@
+class EmbeddedAnsible
+  APPLIANCE_ANSIBLE_DIRECTORY = "/opt/ansible-installer".freeze
+  ANSIBLE_ROLE                = "embedded_ansible".freeze
```

Comment:
Is this knowledge something that should live elsewhere?

Is there a more "official" place where we encode the exact role names?